src/Pages/Basket/Basket.js

```diff
@@ -0,0 +1,321 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import ItemList from './ItemList/ItemList';
+import QuantityForm from './QuantityForm/QuantityForm';
+import API from '../../config';
+import sum from '../../utils/sum';
+import './Basket.scss';
+
+class Basket extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      items: [],
+      checkedItemsId: [],
+      quantityFormId: '',
+      isCheckedAllItems: true,
+      isClickedCountButton: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchBasketItems();
+  }
+
+  fetchBasketItems = async () => {
+    try {
+      const res = await fetch(`${API}/orders/cart`, {
+        method: 'GET',
+        headers: {
+          Authorization: `${localStorage.getItem('access_token')}`,
```

Comment:
```suggestion
          Authorization: getToken(),
```
이렇게 사용할 수 있는 형태로 src/utils에 함수로 만들어주시면 좋을거 같아요

Reply:
네 확인했습니다!